platform-tools-sdk/cargo-build-sbf/src/main.rs

```diff
@@ -1138,8 +1143,8 @@ fn main() {
         .arg(
             Arg::new("arch")
                 .long("arch")
-                .possible_values(["sbfv1", "sbfv2"])
```

Comment:
this is going to break any external consumers. we need to properly deprecate these as aliases by retaining them (ideally hidden) and throwing a warning with migration guidance

Reply:
`sbpfv1` was the default version, so people wouldn't be using `--arch sbpfv1` directly. Also, `sbpfv2` was never divulged as ready to be used, nor have we enabled its deployment in any network.